src/facade/dragonfly_connection.cc

```diff
@@ -1119,7 +1118,8 @@ void Connection::HandleMigrateRequest() {
     this->Migrate(dest);
   }
 
-  DCHECK(dispatch_q_.empty());
+  // This triggers on rueidis SingleIntegrationTest
```

Comment:
I think this comment can be improved. Explain how it happens that `DCHECK(dispatch_q_.empty());` fails after the migration, "rueidis SingleIntegrationTest" is irrelevant here